earth_enterprise/rpms/post-install_fusion.sh

```diff
@@ -0,0 +1,303 @@
+	    echo -e "Consider backing up your asset root. $GEEF will warn you when"
+            echo -e "attempting to run with a non-upgraded asset root."
+            echo -e "$UPGRADE_MESSAGE"
+                
+		# Note: we don't want to do the recursive chown on the asset root unless absolutely necessary
+            $BASEINSTALLDIR_OPT/bin/geconfigureassetroot --fixmasterhost --noprompt  $NOCHOWN --assetroot $ASSET_ROOT
+            $BASEINSTALLDIR_OPT/bin/geupgradeassetroot --noprompt $NOCHOWN --assetroot $ASSET_ROOT
+	    chown -R $GEFUSIONUSER_NAME:$GROUPNAME $ASSET_ROOT
+        fi  
+    fi
+}
+
+fix_postinstall_filepermissions()
+{
+    # Run    
+    chmod 775 $BASEINSTALLDIR_OPT/run
+    chmod 775 $BASEINSTALLDIR_VAR/run
+    chown $ROOT_USERNAME:$GROUPNAME $BASEINSTALLDIR_OPT/run
+    chown $ROOT_USERNAME:$GROUPNAME $BASEINSTALLDIR_VAR/run
+
+    # Logs
+    chmod 775 $BASEINSTALLDIR_VAR/log
+    chmod 775 $BASEINSTALLDIR_OPT/log
+    chown $ROOT_USERNAME:$GROUPNAME $BASEINSTALLDIR_VAR/log
+    chown $ROOT_USERNAME:$GROUPNAME $BASEINSTALLDIR_OPT/log
+
+    # Etc
+    chmod 755 $BASEINSTALLDIR_ETC
+    chmod 755 $BASEINSTALLDIR_OPT/etc
+    chmod 644 $SYSTEMRC
+
+    # Other folders
+    chmod 755 $BASEINSTALLDIR_OPT
+    chmod 755 $BASEINSTALLDIR_VAR
+    chmod -R 755 $BASEINSTALLDIR_OPT/lib
+    chmod -R 555 $BASEINSTALLDIR_OPT/bin
+    chmod 755 $BASEINSTALLDIR_OPT
+    
+    # suid enabled
+    chmod +s $BASEINSTALLDIR_OPT/bin/geserveradmin
+    chmod -R 755 $BASEINSTALLDIR_OPT/qt
+    chmod 755 $BASEINSTALLDIR_ETC/openldap
+
+    # Share
+    find $BASEINSTALLDIR_OPT/share -type d -exec chmod 755 {} \;
+    find $BASEINSTALLDIR_OPT/share -type f -exec chmod 644 {} \;
+    chown -R $ROOT_USERNAME:$ROOT_USERNAME $BASEINSTALLDIR_OPT/share
+    chmod ugo+x $BASEINSTALLDIR_OPT/share/support/geecheck/geecheck.pl
+    chmod ugo+x $BASEINSTALLDIR_OPT/share/tutorials/fusion/download_tutorial.sh
+
+    # Restrict permissions to uninstaller and installer logs
+    chmod -R go-rwx $INSTALL_LOG_DIR
+}
+
+final_assetroot_configuration()
+{
+    if [ $IS_SLAVE == true ]; then
+        $BASEINSTALLDIR_OPT/bin/geselectassetroot --role slave --assetroot $ASSET_ROOT
+    else
+        $BASEINSTALLDIR_OPT/bin/geselectassetroot --assetroot $ASSET_ROOT
+
+	mkdir -p $BASEINSTALLDIR_OPT/share/tutorials
+	$BASEINSTALLDIR_OPT/bin/geconfigureassetroot --addvolume opt:$BASEINSTALLDIR_OPT/share/tutorials --noprompt --nochown
+	if [ $? -eq 255 ]; then
+            echo -e "The geconfigureassetroot utility has failed on attempting"
+            echo -e "to add the volume 'opt:$BASEINSTALLDIR_OPT/share/tutorials'."
+            echo -e "This is probably due to a volume named 'opt' already exists."
+        fi
+    fi
+}
+
+final_fusion_service_configuration()
+{
+    chcon -t texrel_shlib_t $BASEINSTALLDIR_OPT/lib/*so*
+
+    if [ $START_FUSION_DAEMON == true ]; then
+        if [ -f "$FUSIONBININSTALL" ]; then
+            $FUSIONBININSTALL start
```

Comment:
We may want to use the `service` command here.

Reply:
ok